dhis-2/dhis-api/src/main/java/org/hisp/dhis/dataapproval/DataApproval.java

```diff
@@ -54,13 +59,13 @@
  * @author Jim Grace
  */
 public class DataApproval implements Serializable {
-  public static final String AUTH_APPROVE = "F_APPROVE_DATA";
+  public static final String AUTH_APPROVE = F_APPROVE_DATA.toString();
```

Comment:
these strings are only used in tests. I had to draw the line somewhere as this work could go on much longer :) 
The PRD code uses `Authorities`